quobyte-csi-driver/templates/storageclass.yaml

```diff
@@ -0,0 +1,23 @@
+{{ range $storageClass := .Values.quobyte.storageClasses }}
```

Comment:
This is user configuration/cluster specific and does not belong in CSI driver. Please use your custom chart to have these templates.